c/test/legacy/cpp/test_sbp_stdio.cc

```diff
@@ -97,11 +96,12 @@ class SbpStdioTest : public ::testing::Test {
 };
 
 TEST_F(SbpStdioTest, ReadsSbpFiles) {
-  EXPECT_EQ(num_entries_in_file("gnss_data.sbp"), 3);
+  EXPECT_EQ(num_entries_in_file("c/test/legacy/cpp/sbp_data/gnss_data.sbp"), 3);
```

Comment:
Please replace magic number `3` or leave a comment describing why 3 entries in the file are expected.

Comments indicating what this test seeks to validate and documentation of `num_entries_in_file` would also be appreciated

Reply:
I'm just updating the path here. I don't know the context behind the magic numbers in these tests.